kube/account/account-deployment.yaml

```diff
@@ -56,7 +56,10 @@ spec:
                   name: huly-secret
                   key: SERVER_SECRET
             - name: TRANSACTOR_URL
-              value: ws://transactor:3333;ws://localhost:3333
+              valueFrom:
+                configMapKeyRef:
+                  name: huly-config
+                  key: TRANSACTOR_URL
```

Comment:
This should stay unchanged, account pod doesn't need to use ingress to reach transactor service.
TRANSACTOR_URL in config needs to be used by frontend. That change should be done in frontend code, not in deployment manifests. Last time I checked it (front pod) was using localhost which would cause problem when reaching frontend through proxy.